package.json

```diff
@@ -12,21 +12,21 @@
 	"type": "module",
 	"scripts": {
 		"astro": "astro",
-		"build": "astro build",
-		"dev": "astro dev",
-		"preview": "astro preview",
-		"start": "astro dev"
+		"build": "node script/moveServiceWorker.js && astro build",
+		"dev": "node script/moveServiceWorker.js && astro dev",
+		"preview": "node script/moveServiceWorker.js && astro preview",
+		"start": "node script/moveServiceWorker.js && astro dev"
```

Comment:
```suggestion
		"build": "ts-node script/moveServiceWorker.js && astro build",
		"dev": "ts-node script/moveServiceWorker.js && astro dev",
		"preview": "ts-node script/moveServiceWorker.js && astro preview",
		"start": "ts-node script/moveServiceWorker.js && astro dev"
```

Reply:
I'll give it a try

Reply:
Astro didn't play nice, it really wanted to compile the file afterwards when we run `pnpm run dev` and `ts-node` was angry. 
I feel like small scripts like this can benefit from being only `.js`